templates/cloud/private-cloud.html

```diff
@@ -0,0 +1,483 @@
+    <div class="col-4 u-hide--small u-vertically-center u-align--center">
+			{{ image (
+				url="https://assets.ubuntu.com/v1/c5b21ec2-Private+Cloud.svg",
+				alt="Private Cloud",
+				width="233",
+				height="170",
+				hi_def=True,
+				loading="auto"
+				) | safe
+			}}
+    </div>
+  </div>
+</section>
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-4">
+			<h2>How does a private cloud work?</h2>
+			<p>Like other cloud architectures, a private cloud enables on-demand resource provisioning through a self-service portal. This improves developer’s agility and simplifies the data centre management process.</p>
+    </div>
+    <div class="col-8">
+      <p><a href="/openstack/what-is-openstack">Private cloud software</a> manages compute, network and storage resources that are available on individual servers, but are distributed across the entire data centre, aggregates them into pools and uses virtualisation technology to divide them into virtual resources that are then allocated to virtual machines (VMs). By adding a <a href="/kubernetes/what-is-kubernetes">containerisation layer</a> on the top, organisations can also provision containerised workloads on-demand.</p>
+			<p>With the private cloud, all resources are available exclusively to a single entity or a service. They are not shared with other tenants, resulting in an increased resource availability. Moreover, during heavy load periods private cloud can dynamically scale-out, providing the required number of resources. The private cloud software can either be installed on the existing servers in the data centre or a completely new infrastructure built solely for the purpose of the private cloud implementation project.</p>
+		</div>
+	</div>
+</section>
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-4">
+			<h2>Public cloud vs private cloud</h2>
+			<p>In principle, the main difference between private and public clouds is in their tenancy model.</p>
+    </div>
+    <div class="col-8">
+      <p>While private cloud is a single-tenant environment where all resources are available to a single entity or a service, public cloud is a multi-tenant environment where all resources are shared among various tenants (various organisations, services or end users) who typically do not know each other at all.</p>
+			<p>Another significant difference is in the <a href="/engage/cloud-pricing-report-2021">total cost of ownership (TCO)</a>. As public clouds implement so-called pay-as-you-go (PAYG) billing, they tend to be more economical in the short term and on a small scale. However, once the number of workloads continues to grow, their costs grow too, leading to an inflation of TCO. Thus, using a <a href="/openstack">cost-effective private cloud</a> as an extension of the public cloud infrastructure in a <a href="/cloud/hybrid-cloud">hybrid cloud architecture</a> is key to achieving infrastructure cost optimisation.</p>
```

Comment:
The hybrid cloud link points to the new page which doesn't exist yet. If you want to merge this before you'll have to remove this link for now and then make an issue to remind you to add it